InternetHospitalWebUI/src/app/Components/Home/home/home.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+import { HomeImages } from '../../../Models/Temp/HomeImage';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss']
+})
+export class HomeComponent implements OnInit {
+  homeImages: HomeImages[] = [
```

Comment:
Take out this hardcoded array to a constant in a separate file and create new service that returns this constant as an Observable object. This service will fetch real posts data from the server on the future.